bin/fix-dbs-parentage

```diff
@@ -58,29 +62,29 @@ def getMissingParentSummaryUsingBlock(datasetList, fileFix=False, insertFlag=Fal
 
     if insertFlag:
         answer = input("fixing DBS parentage: Are you sure?('y'/'n') - needs ' single quote: ").lower()
+        if answer != 'y':
+            print("Aborting the procedure, Nothing updated")
+            exit(1)
+    else:
+        print("Only testing perfomance: No insert")
+
+    if fileFix:
+        print("Missing parentage fix using missing file checking for datasets")
+    else:
+        print("Missing parentage fix using missing block checking for datasets")
 
     for dataset in datasetList:
         if isinstance(dataset, dict):
             dsName = dataset['dataset']
         else:
             dsName = dataset
 
-        if insertFlag:
-            if answer == 'y':
-                if fileFix:
-                    print("Insert missing parentage using file checking for dataset: ", dataset)
-                    failedBlocks = dbsService.insertMissingParentageForAllFiles(dsName, filterFilesWithParents=fileFix,
-                                                                                insertFlag=insertFlag)
-                else:
-                    print("Insert missing parentage using block checking for dataset: ", dataset)
-                    failedBlocks = dbsService.fixMissingParentageDatasets(dsName)
-            else:
-                print("Aborting the procedure, Nothing updated")
-                exit(1)
-        else:
-            print("Only testing perfomance: No insert")
-            failedBlocks = dbsService.insertMissingParentageForAllFiles(dsName,  filterFilesWithParents=fileFix,
+        if fileFix:
+            failedBlocks = dbsService.insertMissingParentageForAllFiles(dsName, filterFilesWithParents=fileFix,
                                                                         insertFlag=insertFlag)
+        else:
+            failedBlocks = dbsService.fixMissingParentageDatasets(dsName, insertFlag=insertFlag)
```

Comment:
Seangchan, are these two methods `insertMissingParentageForAllFiles` and `fixMissingParentageDatasets` actually writing to DBS? If so, then I'd suggest to move then somewhere else or rename this `getMissingParentSummaryUsingBlock` method name, such that it's clear what's only reading and what is writing to DBS.

Reply:
It only insert to DBS when insertFlag is set to True. I can update the name.

Reply:
Seangchan, I'd suggest then to update this method getMissingParentSummaryUsingBlock to updateParentageInfo or something like that.

Reply:
I will do that. Thanks,